R/reportEDGETransport2.R

```diff
@@ -448,7 +448,7 @@ reportEDGETransport2 <- function(output_folder = ".",
     load_factor <- load_factor[, c("year", "region", "vehicle_type", "loadFactor", "technology")]
     demand_vkm <- merge(demand_km, load_factor, by = c("year", "region", "vehicle_type", "technology"))
     demand_vkm[, value := value / loadFactor] ## billion vkm
-    load_factor <- unique(loadFactor[year >= 1990, c("region", "year", "loadFactor", "vehicle_type")])
+    load_factor <- unique(load_factor[year >= 1990, c("region", "year", "loadFactor", "vehicle_type")])
```

Comment:
Does this mean the vintages reporting now works with both iterative and SA?

Reply:
Yes.. I implemented it already a few days ago but still received an error - turns out it was due to a typo in my bugfix :)